changelog/13638.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fixed deprecated ``gh pr new`` command in :file:`scripts/prepare-release-pr.py`.
+The script now uses ``gh pr create`` which is compatible with GitHub CLI v2.0+.
```

Comment:
```suggestion
Fixed deprecated :command:`gh pr new` command in :file:`scripts/prepare-release-pr.py`.
The script now uses :command:`gh pr create` which is compatible with GitHub CLI v2.0+.
```

Reply:
I got curious because those were new to me, but it doesn't seem `:command:` or `:file:` have any effect other than format the text as code:

<img width="845" height="70" alt="image" src="https://github.com/user-attachments/assets/28a9c89f-54f4-4f05-9786-aac310010b7b" />

I don't mind leaving it there if you prefer, but I mention in case it was expected to produce different results.

Reply:
I think Furo needs to learn to style them. But markup-wise, they are nice to have.

Reply:
https://github.com/pradyunsg/furo/discussions/890